core/storage/suggestion/gae_models.py

```diff
@@ -3568,6 +3548,68 @@ def export_data(
         return user_data
 
 
+T = TypeVar(
+    'T',
+    TranslationSubmitterTotalContributionStatsModel,
+    TranslationReviewerTotalContributionStatsModel,
+    QuestionSubmitterTotalContributionStatsModel,
+    QuestionReviewerTotalContributionStatsModel)
```

Comment:
```suggestion
T = TypeVar(
    'T',
    TranslationSubmitterTotalContributionStatsModel,
    TranslationReviewerTotalContributionStatsModel,
    QuestionSubmitterTotalContributionStatsModel,
    QuestionReviewerTotalContributionStatsModel
)
```

Do not use `T`, give this type variable a more meaningful name.

Reply:
done